### PR TITLE
Add integration installation instructions

### PR DESCRIPTION
**Pull request by vikrammoule-redislabs**
Add a bit more details for new users about installing the community integrations.

### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: redisenterprise/README.md [hunk 1]
```diff
@@ -37,7 +37,23 @@ This integration is provided by Redis Labs.
 
 ## Setup
 
-Copy the [sample configuration][2] and update the required sections to collect data from your Redis Enterprise cluster
+### Installation
+
+If you are using Agent v6.8+ follow the instructions below to install the RedisEnterprise check on your host. See the dedicated Agent guide for [installing community integrations][3] to install checks with the [Agent prior v6.8][4] or the [Docker Agent][5]:
```
Comment (kayayarai) [inline redisenterprise/README.md:42]: The target page for these links has changed quite a bit. See https://docs.datadoghq.com/agent/guide/use-community-integrations Not sure if "v6.8" should be changed to "v7.21/v6.21" in this sentence. but the link targets for links 3, 4, and 5 are: 
- https://docs.datadoghq.com/agent/guide/use-community-integrations/
- https://docs.datadoghq.com/agent/guide/use-community-integrations/?tab=agentearlierversions
- https://docs.datadoghq.com/agent/guide/use-community-integrations/?tab=docker

**Review by kayayarai — CHANGES_REQUESTED** [review]
Some updates to the links you're using, but otherwise LGTM

**Comment by vikrammoule-redislabs** [comment]
Thanks, @kayayarai.
I agree with your suggestion. Do you need me to submit a new PR?

**Comment by kayayarai** [comment]
> Thanks, @kayayarai.
> I agree with your suggestion. Do you need me to submit a new PR?

No, you can just make another commit/push to your branch, and it will update this PR.

**Review by sarah-witt — APPROVED** [review]
Looks good from the integrations side!

**Pull request merged**